tasks/rubocop.rake

```diff
@@ -0,0 +1,23 @@
+module Rubocop
+  module ProjectExtension
+    include Extension
+
+    desc 'Run RuboCop on select files'
+      require 'rubocop/rake_task'
+      RuboCop::RakeTask.new(:rubocop) do |task|
+      task.verbose = true
+      task.patterns =
```

Comment:
I would make these patterns project specific.  Something like `File.join(project.base_dir, '**/*.rb')` as a default.  Go ahead and create a `Config` class so that we can set options in `buildfile`.  See `msgfmt.rake`, `gettext.rake`, or `liquibase.rake` for examples of setting up configuration objects for tasks.